Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```diff
@@ -145,28 +150,28 @@ configuration:
   display: Key File as Text
   name: key
   type: 4
-  hidden: true
   required: false
+  hidden: true
 - additionalinfo: Set the number of stix object that will be requested with each taxii poll (http request). A single fetch is made of several taxii polls. Changing this setting can help speed up fetches, or fix issues on slower networks. Please note server restrictions may apply, overriding and limiting the "requested limit".
   defaultvalue: '100'
   display: Max STIX Objects Per Poll
   name: limit_per_request
   type: 0
   required: false
-- additionalinfo: Choose how to handle complex observations. Two or more Observation Expressions MAY be combined using a complex observation operator such as "AND", "OR". e.g. `[ IP = 'b' ] AND [ URL = 'd' ]`
-  defaultvalue: Skip indicators with more than a single observation
+- additionalinfo: "Choose how to handle complex observations. Two or more Observation Expressions MAY be combined using a complex observation operator such as \"AND\", \"OR\". e.g. `[ IP = 'b' ] AND [ URL = 'd' ]`"
```

Comment:
Recommend:
"Choose how to handle complex observations. Two or more observation expressions may be combined using a complex observation operator such as \"AND\" or \"OR\". For example:  `[ IP = 'b' ] AND [ URL = 'd' ]`"